css/templates/land-see/page.css

```diff
@@ -712,3 +730,145 @@ h4,
     transform: translate(-50%, 0);
   }
 }
+
+/********** Stories Template **********/
+
+.land-see-category h1 {
+  font: 500 var(--h1)/var(--line-height-4) var(--font-primary);
+  color: var(--color-font-primary);
+}
+
+.land-see-categories-nav-page-load,
+.land-see-categories-nav {
+  overflow: hidden;
+  height: 35px;
+  border-bottom: solid 1px var(--color-story-navigation-border);
+}
+
+.land-see-categories-button {
+  font: 500 var(--h6)/var(--line-height-1) var(--font-navigation);
+  text-transform: uppercase;
+  letter-spacing: .1rem;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  padding: 0 0 3px 0;
+  cursor: pointer;
+}
+
+.land-see-categories-button:focus,
+.land-see-next-page:focus,
+.land-see-prev-page:focus  {
+  outline: 0;
+}
+
+.land-see-selected-category,
+.land-see-categories-nav-page-load button:target {
+  border-bottom: solid 1px var(--color-story-navigation-accent);
+}
+
+.land-see-paging {
+  border-top: solid 1px var(--color-story-navigation-border);
+}
+
+.land-see-next-page,
+.land-see-prev-page {
+  font: 400 var(--h7)/var(--line-height-4) var(--font-navigation);
+  color: var(--color-font-accent);
+  letter-spacing: 1.4px;
+  border: none;
+  text-transform: uppercase;
+  text-decoration: underline;
+  background-color: transparent;
+  padding: 0 0 3px 0;
+  cursor: pointer
+}
+
+.land-see-next-page:after,
```

Comment:
::after, ::before - change everywhere - https://stackoverflow.com/questions/8980475/after-vs-after

Reply:
What, we're not worried about supporting IE8? :P

Reply:
No , Amp does not support 8 :) only latest 2 versions of any browser